standard/documentation-comments.md

```diff
@@ -1015,33 +1012,15 @@ namespace Graphics
     /// </summary>
     public class Point
     {
-        /// <summary>
```

Comment:
We either shouldn't change this, or we should remove the field documentation from the XML below - as it stands, the change invalidates the example.

Reply:
Good catch, Jon! I changed the XML to reflect the removal of the 2 named fields.